PocoDoc/cfg/mkdoc-poco.xml

```diff
@@ -7,7 +7,7 @@
 				${PocoBuild}/*/include/Poco/*/*/*.h
 				${PocoBuild}/*/include/Poco/*/*.h
 				${PocoBuild}/*/*/include/Poco/*/*/*.h
-				${PocoBuild}/Data/src/sql-parser/*.h
+				${PocoBuild}/Data/SQLParser/*.h
```

Comment:
Is it necessary to have SQL parser includes for user documentation generation?

Reply:
Probably best to remove it too